specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9155 @@
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ScanRulesetType",
+            "modelAsString": true
+          }
+        },
+        "scanLevelType": {
+          "enum": [
+            "Full",
+            "Incremental"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ScanLevelType",
+            "modelAsString": true
+          }
+        },
+        "errorMessage": {
+          "type": "string",
+          "readOnly": true
+        },
+        "error": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          ],
+          "readOnly": true
+        },
+        "runType": {
+          "type": "string",
+          "readOnly": true
+        },
+        "dataSourceType": {
+          "enum": [
+            "None",
+            "AzureSubscription",
+            "AzureResourceGroup",
+            "AzureSynapseWorkspace",
+            "AzureSynapse",
+            "AdlsGen1",
+            "AdlsGen2",
+            "AmazonAccount",
+            "AmazonS3",
+            "AmazonSql",
+            "AzureCosmosDb",
+            "AzureDataExplorer",
+            "AzureFileService",
+            "AzureSqlDatabase",
+            "AmazonPostgreSql",
+            "AzurePostgreSql",
+            "SqlServerDatabase",
+            "AzureSqlDatabaseManagedInstance",
+            "AzureSqlDataWarehouse",
+            "AzureMySql",
+            "AzureStorage",
+            "Teradata",
+            "Oracle",
+            "SapS4Hana",
+            "SapEcc",
+            "PowerBI"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "DataSourceType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Scan": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "AzureSubscriptionCredential",
+            "AzureSubscriptionMsi",
+            "AzureResourceGroupCredential",
+            "AzureResourceGroupMsi",
+            "AzureSynapseWorkspaceCredential",
+            "AzureSynapseWorkspaceMsi",
+            "AzureSynapseCredential",
+            "AzureSynapseMsi",
+            "AdlsGen1Credential",
+            "AdlsGen1Msi",
+            "AdlsGen2Credential",
+            "AdlsGen2Msi",
+            "AmazonAccountCredential",
+            "AmazonS3Credential",
+            "AmazonS3RoleARN",
+            "AmazonSqlCredential",
+            "AzureCosmosDbCredential",
+            "AzureDataExplorerCredential",
+            "AzureDataExplorerMsi",
+            "AzureFileServiceCredential",
+            "AzureSqlDatabaseCredential",
+            "AzureSqlDatabaseMsi",
+            "AmazonPostgreSqlCredential",
```

Comment:
from diff, it seems "AmazonPostgreSqlRoleARN" is removed?

Reply:
Yes, `AmazonPostgreSqlRoleARN` is removed as part of a schema migration